docs/guides/developer/docs/release-manager.md

```diff
@@ -19,6 +19,26 @@ meetings. It is important to note that, while release managers drive the release
 charge of both the work and the decision making, meaning that votes and successful proposals from this body take
 precedence over release manager decisions.
 
+Recommended practices for release managers
+------------------------------
+
+This is a recommendation of best practices to help to organize the duties of the release managers after they have been choosen
+these recomendations came from the experience of past release managers.
+
+### After the election of the release manager
```

Comment:
Would you want to do that this early? I would probably not start before creating the release branch since that is the time you kind of know what's going in the release, and you still have quite a bit of time left before the release.

Reply:
Initially, I was thinking after the release cut, but when you have a lot of PRs, it is more difficult to follow them, i'm more into the idea to create the drafts to have  it ready to add/modify/remove the changes month by month basis. 

In the case of WordPress draft. Do you think is too early to make it? Because it's very short and you don't need to prepare it like the release notes.

Reply:
The wordpress can probably wait, but the initial draft PR for the release notes is a good idea.  Bonus points if we can remember to discuss, weekly, the features which made it in and thus should be added to said document.  My original idea with having folks put snippets into the `releasenotes` directory was that I *could not keep track* of what went in week-to-week, and thus would miss things.  The less we have a single person looking at this, or the more regimented we can become with respect to making sure that document is up to date at all times, the better IMO.